deploy/scripts/install.sh

```diff
@@ -21,6 +21,12 @@ Options:
     Use specified name for the resource group instead of default one.
   --node-env <node environment>
     Forces the use of specific node environment as the application runtime setting (default: development).
+  --kube-environment-id <kube environment id>
+    Provide the kube environment id to deploy to Arc
+  --custom-location-id <custom location id>
+    Provide the custome location id to deploy to Arc
+  --data-controller-id <data controller id>
+    Provide the ResourceId of the Data Controller
```

Comment:
It would be helpful to mention that these are only needed if deploying to Arc.

Reply:
Like this?